### PR TITLE
Support for geo_shape filters.

### PR DESCRIPTION
**Pull request by henningp**

### CODE REVIEW

**Comment by sksamuel** [comment]
This is great. Do you think you could get this into 2.0.0 on master too ?

**Comment by henningp** [comment]
I'll give it a shot tomorrow.

File: project/Build.scala [hunk 1]
```diff
@@ -35,8 +35,9 @@ object Build extends Build {
       "org.slf4j" % "log4j-over-slf4j" % Slf4jVersion % "test",
       "org.mockito" % "mockito-all" % MockitoVersion % "test",
       "org.scalatest" %% "scalatest" % ScalatestVersion % "test",
-      "org.codehaus.groovy" % "groovy" % GroovyVersion % "test"
-
+      "org.codehaus.groovy" % "groovy" % GroovyVersion % "test",
+      "org.codehaus.groovy" % "groovy" % GroovyVersion % "test",
```
Comment (martin-g) [inline project/Build.scala]: It seems the dependency for Groovy is doubled.

File: project/Build.scala [hunk 2]
```diff
@@ -35,8 +35,9 @@ object Build extends Build {
...
+      "com.vividsolutions" % "jts" % "1.13" % "test"
```
Comment (martin-g) [inline project/Build.scala]: I don't see any new imports which should come from this new dependency.
Did I miss something?
Maybe it is an `optional` dependency in ElasticSearch ?!

Comment (henningp) [inline project/Build.scala]: Yes, exactly.

**Pull request merged**